### ISSUE TITLE
Getting error "Cannot set property 'permissionLevel' of undefined" when switching folders

### ISSUE THREAD

**Issue by amiram** [OfficeDev/office-js#1026]
When switching folders and my add-in is pinned, I see an error from office.js library
`Cannot set property 'permissionLevel' of undefined`

It is coming from these lines, source map indicates it is https://appsforoffice.microsoft.com/lib/1.1/hosted/outlook-web-16.01.alpha.debug.js:

```
    this.initialize = function (data) {
      if (data === null) { // Should check not only for null. Should be if(!data)
        recreateAdditionalGlobalParametersSingleton(true);
        _this.initialData = null;
        _this.item = null;
      } else {
        _this.initialData = data;
// _this.initialData is undefined since data is undefined
        _this.initialData.permissionLevel = calculatePermissionLevel();
        _this.item = prepareApiSurface_getItem();
        var supportsAdditionalParameters = getAppName() !== OutlookAppOm_OSF.AppName.Outlook || isOutlook16_isOutlook16OrGreater(getInitialDataProp("hostVersion")) || isApiVersionSupported("1.5", _this.officeAppContext);
        recreateAdditionalGlobalParametersSingleton(supportsAdditionalParameters);

        if (typeof data.itemNumber !== "undefined") {
          getAdditionalGlobalParametersSingleton().setCurrentItemNumber(data.itemNumber);
        }
      }
    };


```
## Steps to Reproduce, or Live Example
1. Open a mail item with a sidebar pinned.
2. Click on a different folder.

## Your Environment

* Platform" Office Online
* Host: Outlook
* Operating System: Ubuntu
* Browser (if using Office Online): Chrome

**Comment by exextoc** [comment]
Thanks for the reporting this issue. It has been put on our backlog. We unfortunately have no timelines to share at this point.

**Comment by exextoc** [comment]
We have fixed the bug and the fix should be available in build 16.0.12419.10000 or later of outlook-web.js file.

**Comment by lindalu-MSFT** [comment]
Closing, issue is fixed. Please reactivate if you need further assistance! Thanks, linda
